packages/local-environment/.env.example

```diff
@@ -5,8 +5,18 @@ PRIVATE_KEY_3=3000000000000000000000000000000000000000000000000000000000000003
 PRIVATE_KEY_4=4000000000000000000000000000000000000000000000000000000000000004
 PRIVATE_KEY_5=5000000000000000000000000000000000000000000000000000000000000005
 PRIVATE_KEY_6=6000000000000000000000000000000000000000000000000000000000000006
-PRIVATE_KEY_7=6000000000000000000000000000000000000000000000000000000000000006
-PRIVATE_KEY_8=6000000000000000000000000000000000000000000000000000000000000006
-PRIVATE_KEY_9=6000000000000000000000000000000000000000000000000000000000000006
+PRIVATE_KEY_7=7000000000000000000000000000000000000000000000000000000000000007
+PRIVATE_KEY_8=8000000000000000000000000000000000000000000000000000000000000008
+PRIVATE_KEY_9=9000000000000000000000000000000000000000000000000000000000000009
 
-AGENTS_IMAGE=gcr.io/nomad-xyz/nomad-agent:prestwich-remove-deploy-gas
\ No newline at end of file
+AGENTS_IMAGE=gcr.io/nomad-xyz/nomad-agent:prestwich-remove-deploy-gas
+
+WETH_ADDRESS=
+
+# Only set the following if you're forking mainnet
+
+ALCHEMY_API_KEY=UEz-5kiUuyGqZhi8SNUrgJrEy-LPqkZK
```

Comment:
@xBalbinus Just heads up that we should remove this api key and rotate it too just in case

Reply:
Got it np